content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/100-cyclic-referential-actions.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Cyclic referential actions with Microsoft SQL Server'
+metaTitle: 'Cyclic referential actions with Microsoft SQL Server'
+metaDescription: 'Circular references can occur between models when each model references another creating a closed loop. Since Microsoft SQL Server does not handle these situations out of the box, learn how to solve this problem.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Circular references can occur between models when each model references another, creating a closed loop. Most SQL databases handle these references out of the box. 
+
+With Microsoft SQL Server, however, if the [referential actions](./) on the relation field is set to something other than `NoAction`, the data model validator will throw an error the cyclic relation.
+
+This is dependent on whether or not the underlying scalar fields are set to be optional or required. If no referential action is specified on the relation, the default values are used:
+
+| Clause     | All of the scalar fields are optional | At least one scalar field is required |
+| :--------- | :----------------------- | :------------------------------------ |
+| `onDelete` | `SetNull`                | `NoAction`                            |
+| `onUpdate` | `Cascade`                | `Cascade`                             |
+
+</TopBlock>
+
+## Self relation
+
+The following model describes a self relation where the chicken references itself with fields that represent both the mother and father chickens.
```

Comment:
instead of "chicken" is should probably be "`Chicken`"

Reply:
I also needed to read that a few time to get the connection between the sentence and the example: the example also uses different terms than "mother" and "father", so using the same terminology or adding that here would help understand the example initially